docs/DG_Diagrams/DineSequence.puml

```diff
@@ -0,0 +1,44 @@
+@startuml
+actor user
+box Logic
+participant ":Duke"
+participant ":Parser"
+participant ":DineCommand"
+end box
+box Ui
+participant "<<class>>\n:Ui"
+end box
+
+user -> ":Duke"
+box Logic
+":Duke" -> ":Parser": user command
+activate ":Parser"
+create ":DineCommand"
+":Parser" -> ":DineCommand": user input
+activate ":DineCommand"
+":DineCommand" --> ":Parser"
+deactivate ":DineCommand"
+":Parser" -> ":DineCommand": executeCommand()
+activate ":DineCommand"
+alt user input empty
+    ":DineCommand" --> ":Parser": invalid input
+else user input not empty
+    ":DineCommand" -> ":DineCommand": checkFaculty()
+    activate ":DineCommand"
+    alt displaycount = 0
+        ":DineCommand" --> ":DineCommand": no match found
+    else displaycount > 0
+        ":DineCommand" -> "<<class>>\n:Ui" : printDineResult()
+        activate "<<class>>\n:Ui"
+        "<<class>>\n:Ui" --> ":DineCommand"
+        deactivate "<<class>>\n:Ui"
+        ":DineCommand" --> ":DineCommand": results
+    deactivate ":DineCommand"
+    end
+    ":DineCommand" --> ":Parser": list of dining options
+deactivate ":DineCommand"
+end
+":Parser" --> ":Duke"
+deactivate ":Parser"
+":Duke" --> user
```

Comment:
![Screenshot_2](https://user-images.githubusercontent.com/59549150/97514088-df926900-19c8-11eb-980b-578aa748fe39.png)

Should there be return arrows here, since invalid input and no match found will return it earlier?